src/components/table/sort.ts

```diff
@@ -11,6 +19,23 @@ export function descendingComparator<T>(a: T, b: T, orderBy: keyof T): 1 | -1 |
   return 0;
 }
 
+function descendingNumComparator<T>(a: T, b: T, orderBy: keyof T): 1 | -1 | 0 | null {
+  const aNumValue = parseFloat((a[orderBy] ?? '0.0') as string);
+  const bNumValue = parseFloat((b[orderBy] ?? '0.0') as string);
+  if (!isNaN(aNumValue) && !isNaN(bNumValue)) {
+    if (bNumValue < aNumValue) {
+      return -1;
+    }
+    if (bNumValue > aNumValue) {
+      return 1;
+    }
+
+    return 0;
```

Comment:
a shortcut for numerical comparison is (bNumValue - aNumValue)

Reply:
argh 
<img width="670" alt="image" src="https://user-images.githubusercontent.com/114949086/196490788-26cc0499-a322-4ce7-b7e2-5ed5cbf39014.png">

Reply:
I might just leave as is instead of fighting with the typechecker :/

Reply:
does not have to be -1
has to be negative for less than, zero for same, positive for greater than

short notation for numerical sort is quite common..

for reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

<img width="808" alt="Screen Shot 2022-10-18 at 9 41 14 AM" src="https://user-images.githubusercontent.com/1865174/196491607-95d1373e-98a4-462c-b83b-09893e2f6737.png">

Reply:
> I might just leave as is instead of fighting with the typechecker :/

sure, that's fine.

Reply:
I'll have to change the return type for `descendingComparator` to `number`, but i can give that a try and make sure the sort still works in the table 👍

Reply:
Oh, it should I guess. `stableSort` already expects `(a: T, b: T) => number` 😅

Reply:
> I'll have to change the return type for `descendingComparator` to `number`, but i can give that a try and make sure the sort still works in the table 👍

Yikes, just saw the type in descendingComparator. Did not intend for my suggestion to generate a lot of change.
Don't bother about this change for now - what you have is good and correct.

Reply:
ah, read this too late 😂 It's okay I already have the change in and tested it

Reply:
Thanks so much!